### DISCUSSION TITLE
Add a Mic Test Function

### DISCUSSION THREAD

**Discussion by Jlobblet** [FakeFishGames/Barotrauma#4672] [Ideas and suggestions]
Barotrauma should have an option to test whether your mic is working ingame.
It probably belongs on the voice chat settings tab, below the current settings there.
![image](https://user-images.githubusercontent.com/37539527/104016419-3331e780-51ae-11eb-9bc7-19f0a2b12e1c.png)
Adding this feature would help users check their mics are working and what they sound like (including fine-tuning volume) within the game - after all, checking your mic is working with a different program doesn't mean it's working the same in Barotrauma.

**Comment by Krzeszny** [comment]
This is really annoying when setting up a voice changer.
Also why is the scale 0-1000% ? What kind of scale is that?

Yesterday:
"Can you hear me"
*no answer*
*tweaking volume*
"Can you hear me now?"
"Move your mic out of your mouth!"
*tweaking volume*
"Can you hear me?"
"Are you using some voice changing software from the 1980's?"
"But can you hear me?"
*no answer*

**Comment by mygamingaccount** [comment]
If crewmates don't complain that my voice is too loud, they complain that it isn't loud enough.
I never know if the sound I'm transmitting is clear or clipping like a MesaBoogie Triple Rectifier on the red channel
